src/redux/reducers/trades.reducer.js

```diff
@@ -83,9 +87,11 @@ const reducer = (state = INITIAL_STATE, action = {}) => {
 
       return {
         ...state,
-        [symbol]: {
-          ...prevData,
-          [id]: data,
+        [chanId]: {
+          [symbol]: {
+            ...prevData,
+            [id]: data,
+          },
```

Comment:
can we now remove symbol  and keep state like
 ```
[chanId]: {
     ...prevData,
     [id]: data,
 }
```

Reply:
@bhoomij sure, updated this way